docs/source/features/parallelisms.rst

```diff
@@ -156,6 +184,40 @@ The NeMo implementation of PP leverages functionalities from Megatron Core. For
 
 For more detailed API usage and configurations related to PP, visit the `Megatron Core Developer Guide <https://docs.nvidia.com/Megatron-Core/developer-guide/latest/api-guide/tensor_parallel.html>`_.
 
+Expert Parallelism
+^^^^^^^^^^^^^^^^^^
+Expert Parallelism (EP) is a type of model parallelism that distributes experts of an MoE across GPUs.
+Unlike other model-parallel techniques, EP is applied to only the expert layers thus does not impact the parallel mapping of the rest of layers.
+
+.. image:: ../nlp/nemo_megatron/images/ep.png
+    :align: center
+    :width: 800px
+    :alt: Expert Parallelism
+
+Enable Expert Parallelism
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable EP, set ``model.expert_model_parallel_size`` to the desired expert parallel size. For example, if the model has six experts (``model.num_moe_experts=6``), then setting ``model.expert_model_parallel_size=3`` results in each GPU processing two experts. The number of experts should be divisible by the expert parallel size.
+
+   .. code-block:: yaml
+
+       expert_model_parallel_size: 3  # Set EP to 3
+
+For further information on configuration, refer to the following documentation: `NeMo Megatron GPT Config <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/conf/megatron_gpt_config.yaml#L68>`_.
+
+
+Implement Expert Parallelism
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The NeMo implementation of Expert Parallelism uses functionality from Megatron Core. Please consult the `Megatron Core MoE layer <https://github.com/NVIDIA/Megatron-LM/blob/e2ec14ab5690fead7e33760b0f8fb20c83b4fd1f/megatron/core/transformer/moe/moe_layer.py#L29>`_ for more MoE implementation details.
+
+
+Activation Partitioning
+-----------------------
+
+In LLM training, a large memory space is needed to store the input activations of the network layers.
+NeMo provides effective activation distribution methods, which is critical in training LLM with a large sequence length or large per-GPU micro-batch size.
+
```

Comment:
Edits to Line 224

Sequence Parallelism (SP) extends tensor-level model parallelism by distributing the computing load and activation memory across multiple GPUs along the sequence dimension of transformer layers.